ui/src/main/webapp/src/app/project/migration-project-form.component.ts

```diff
@@ -49,6 +50,10 @@ export class MigrationProjectFormComponent extends FormComponent implements OnIn
             }
 
             this.isInWizard = flatRouteData.data.hasOwnProperty('wizard') && flatRouteData.data['wizard'];
+            this.isCreating = flatRouteData.data.hasOwnProperty('creating') && flatRouteData.data['creating'];
+
+            if(this.isCreating)
+                this._migrationProjectService.deleteProvisionalProjects().subscribe(res => {});
```

Comment:
I think that there is a race condition here. It shouldn't let you edit/save the form until this completes.